src/routes/communityGroups/community_groups.router.js

```diff
@@ -0,0 +1,67 @@
+const express = require("express");
+const { check, body } = require("express-validator");
+
+const {
+  listCommunityGroups,
+  getCommunityGroup,
+  addCommunityGroup,
+  updateCommunityGroup,
+  deleteCommunityGroup
+} = require("./community_groups.contoller");
```

Comment:
Can you think of a way to make the names of these methods just the 'action' being taken (e.g. list, get, add etc), without losing the context of `CommunityGroup`?

Reply:
Like so? (in `community_groups.contoller`)
```
module.exports = {
  list: listCommunityGroups,
  getSingle: getCommunityGroup,
  add: addCommunityGroup,
  update: updateCommunityGroup,
  deleteItem: deleteCommunityGroup
};
```

Or like so (in `community_groups.contoller`)
```
const {
  listCommunityGroups: list,
  getCommunityGroup: getSingle,
  addCommunityGroup: add,
  updateCommunityGroup: update,
  deleteCommunityGroup: deleteItem
} = require("./community_groups.contoller");
```

(But not both)

Reply:
The first one is what I had in mind.

You could take it further by naming all of the controller methods just their action, so instead of 'listCommunityGroups', it's just be named 'list'. Then you don't have to do extra stuff on the import, and you can rely on the context of the file to indicateon which resource the endpoints operate.

I noticed that your examples call `delete` `deleteItem`. I would assume this is because `delete` is a reserved word in javascript. What's an alternative verb (english verb, not HTTP) to 'delete' you could use instead?

Reply:
Probably `remove`.

I'll admit I just changed those values for conversation sake, not in actual code.
Style-wise, I haven't given too much thought to whether I'd rather have the short-names-with-context or the complete-fully-descriptive-long-names.

Reply:
Good! This still might seem trivial, but it can go a long way towards making your codebase easier to maintain.

Above all though, consistency is key. You get to decide if you want short names with context or fully descriptive ones.  I'd put some thought into that sooner rather than later because as the project grows it'll get harder to ensure everything is consistent.